src/visitors/components/SignInActivitiesGrid.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Row, Col } from 'react-flexbox-grid';
+import SignInActivitiesGridItem from './SignInActivitiesGridItem';
+import { pairs } from '../../util';
+
+
+const MarginSpacedCol = styled(Col)`
+  margin: 1em 0;
+`;
+
+const SignInActivitiesGrid = props => (
+  <>
```

Comment:
I've not seen this before, is it okay to give an unnamed element to jsx?

Reply:
It's short-hand for `<Fragment>`